SPECS/lldb/lldb.spec

```diff
@@ -56,7 +58,7 @@ cmake -DCMAKE_INSTALL_PREFIX=/usr           \
       -DCMAKE_BUILD_TYPE=Release            \
       -DLLDB_PATH_TO_LLVM_BUILD=%{_prefix}  \
       -DLLDB_PATH_TO_CLANG_BUILD=%{_prefix} \
-      -DLLVM_DIR=/usr/lib/cmake/llvm        \
+      -DLLVM_DIR=%{_libdir}/cmake/llvm        \
       -DLLVM_BUILD_LLVM_DYLIB=ON ..         \
       -DLLDB_DISABLE_LIBEDIT:BOOL=ON        \
       -DPYTHON_EXECUTABLE:STRING=%{__python2} \
```

Comment:
These cmake args still use python2- these should be changed to python3

Reply:
```suggestion
      -DPYTHON_EXECUTABLE:STRING=%{__python3} \
```